src/algebra/ring/aut.lean

```diff
@@ -60,4 +61,32 @@ by refine_struct { to_fun := ring_equiv.to_mul_equiv }; intros; refl
 def to_perm : ring_aut R →* equiv.perm R :=
 by refine_struct { to_fun := ring_equiv.to_equiv }; intros; refl
 
+/-- The tautological action by the group of automorphism of a ring `R` on `R`.-/
```

Comment:
```suggestion
/-- The tautological action by the group of automorphism of a ring `R` on `R`.

This generalizes `function.End.apply_mul_action`. -/
```
You should also modify the docstring for `function.End.apply_mul_action` to include this new definition.